content/_help/verify-your-identity/verify-your-address-by-mail._fr.md

```diff
@@ -25,4 +25,4 @@ Pour demander un nouveau code de vérification :
 2. Sélectionnez « Envoyez-moi un autre courrier » lorsque vous êtes invité à saisir votre code.
 
 ## Articles Liés
-- [Vérifier votre identité](/fr/help/verify-your-identity/how-to-verify-your-identity/)
\ No newline at end of file
+- [Vérifier votre identité](/fr/help/verify-your-identity/how-to-verify-your-identity/)
```

Comment:
There was no newline at the end of the file, and my editor *insisted* on putting one in, which is what this (confusing) diff is.